guides/plugins/apps/local-development/app-development-with-docker.md

```diff
@@ -0,0 +1,200 @@
+# App development with docker
+
+## Overview
+
+This guide will walk you through the process of combining Shopware and your app in one setup.
+
+## File Structure
+
+At first, you should clone the app template from [GitHub](https://github.com/shopwareLabs/AppTemplate) and create a `manifest.xml` for your app. Or take a glance at our fully working example based on this template: [appExample](https://github.com/shopwareLabs/AppExample).  
```

Comment:
```suggestion
At first, you need to clone the app template from [GitHub](https://github.com/shopwareLabs/AppTemplate) and create a `manifest.xml` for your app. Or take a glance at our fully working example based on this template: [appExample](https://github.com/shopwareLabs/AppExample).  
```
This is a manual, as such it needs to be assertative. Words like "should" should be avoided, unless there are legitimate alternatives or it describes the result of a third party systems. E.g "change your dns record and your website **should** be reachable from that domain"